src/update/UpdateManager.cpp

```diff
@@ -189,6 +185,7 @@ void UpdateManager::GetUpdates(bool ignoreNetworkErrors)
                                 
                                 });
 
+                        if(mProgressDialog != nullptr)
```

Comment:
Yes, null check now avoids the crash.

But maybe the progress dialog should be created more than once, not in call_once, so the destructions and creations match.

Reply:
Destructions and creations is already match, because `setRequestFinishedCallback` with progress dialog desruction called once, only when downloading is finished.

Reply:
But my point was that using call_once isn't right.  Maybe the dialog will be created more than once, in unusual but possible circumstances.  Maybe download fails once, but is attempted again.